libnavigation-base/src/main/java/com/mapbox/navigation/base/trip/model/eh/EHorizonMapper.kt

```diff
@@ -128,6 +128,8 @@ internal fun RoadObjectDistanceInfo.mapToRoadObjectDistanceInfo(
             pointDistanceInfo.distance
         )
         isPolygonDistanceInfo -> with(polygonDistanceInfo) {
+            val distanceToNearestEntry = if (entrances.isNotEmpty()) entrances[0].distance else 0.0
+            val distanceToNearestExit = if (exits.isNotEmpty()) exits[0].distance else 0.0
```

Comment:
@korshaknn @mskurydin @etl @SiarheiFedartsou Do we want to expose `entrances` and `exits`? 🤔

Reply:
Yeah, we want to expose them. There are some cases when it isn't enough to have only one entry/exit for an object

Reply:
Do we want then to expose publicly `SubGraphDistanceInfo` entirely?

Reply:
Gate contains probability and position as well, I'm sure they should be exposed. Also some other fields might be added in the future release